deploy-manage/security/self-auto-setup.md

```diff
@@ -0,0 +1,149 @@
+* [Download](https://www.elastic.co/downloads/kibana) and unpack the `kibana` package distribution for your environment.
+
+
+## Start {{es}} and enroll {{kib}} with security enabled [stack-start-with-security]
+
+1. From the installation directory, start {{es}}.
+
+    ```shell
+    bin/elasticsearch
+    ```
+
+    The command prints the `elastic` user password and an enrollment token for {{kib}}.
+
+2. Copy the generated `elastic` password and enrollment token. These credentials are only shown when you start {{es}} for the first time.
+
+    ::::{note}
+    If you need to reset the password for the `elastic` user or other built-in users, run the [`elasticsearch-reset-password`](elasticsearch://reference/elasticsearch/command-line-tools/reset-password.md) tool. To generate new enrollment tokens for {{kib}} or {{es}} nodes, run the [`elasticsearch-create-enrollment-token`](elasticsearch://reference/elasticsearch/command-line-tools/create-enrollment-token.md) tool. These tools are available in the {{es}} `bin` directory.
+
+    ::::
+
+
+    We recommend storing the `elastic` password as an environment variable in your shell. Example:
+
+    ```sh
+    export ELASTIC_PASSWORD="your_password"
+    ```
+
+3. (Optional) Open a new terminal and verify that you can connect to your {{es}} cluster by making an authenticated call.
+
+    ```shell
+    curl --cacert config/certs/http_ca.crt -u elastic:$ELASTIC_PASSWORD https://localhost:9200
+    ```
+
+4. From the directory where you installed {{kib}}, start {{kib}}.
+
+    ```shell
+    bin/kibana
+    ```
+
+5. Enroll {{kib}} using either interactive or detached mode.
+
+    * **Interactive mode** (browser)
+
+        1. In your terminal, click the generated link to open {{kib}} in your browser.
+        2. In your browser, paste the enrollment token that you copied and click the button to connect your {{kib}} instance with {{es}}.
+
+            ::::{note}
+            {{kib}} won’t enter interactive mode if it detects existing credentials for {{es}} (`elasticsearch.username` and `elasticsearch.password`) or an existing URL for `elasticsearch.hosts`.
+
+            ::::
+
+    * **Detached mode** (non-browser)
+
+        Run the `kibana-setup` tool and pass the generated enrollment token with the `--enrollment-token` parameter.
+
+        ```sh
+        bin/kibana-setup --enrollment-token <enrollment-token>
+        ```
+
+## Enroll additional nodes in your cluster [stack-enroll-nodes]
+
+:::{include} /deploy-manage/deploy/self-managed/_snippets/enroll-nodes.md
+:::
+
+## Connect clients to {{es}} [_connect_clients_to_es_5]
+
+:::{include} /deploy-manage/deploy/self-managed/_snippets/connect-clients.md
+:::
+
+### Use the CA fingerprint [_use_the_ca_fingerprint_5]
+
+:::{include} /deploy-manage/deploy/self-managed/_snippets/ca-fingerprint.md
+:::
+
+### Use the CA certificate [_use_the_ca_certificate_5]
+
+:::{include} /deploy-manage/deploy/self-managed/_snippets/ca-cert.md
```

Comment:
just noticed that some of these snippets are using `slash` and `es-conf` vars that need to be set at the page level for them to render

<img width="926" alt="image" src="https://github.com/user-attachments/assets/5152202f-ca44-4cd0-88c7-3353aff6c12b" />

Reply:
good catch, thanks! solved now